pkg/compactor/compactor.go

```diff
@@ -43,8 +43,10 @@ const (
 )
 
 const (
-	blocksMarkedForDeletionName = "cortex_compactor_blocks_marked_for_deletion_total"
-	blocksMarkedForDeletionHelp = "Total number of blocks marked for deletion in compactor."
+	blocksMarkedForDeletionName        = "cortex_compactor_blocks_marked_for_deletion_total"
+	blocksMarkedForDeletionHelp        = "Total number of blocks marked for deletion in compactor."
+	partialBlocksMarkedForDeletionName = "cortex_compactor_partial_blocks_marked_for_deletion_total"
+	partialBlocksMarkedForDeletionHelp = "Total number of partial blocks marked for deletion in compactor."
```

Comment:
We don't need these constants. For `blocksMarkedForDeletionName` we have a constant because metric is created in two places. That's not the case for `partialBlocksMarkedForDeletion` counter, we can just inline the strings there.